kf_lib_data_ingest/etl/transform/transform.py

```diff
@@ -263,4 +359,12 @@ def _run(self, data_dict):
             self.logger.warning('Skipping null processing because no target '
                                 'schema was found')
 
+        # Source to target ID translation
+        id_cache = read_json(self.id_cache_filepath, default={})
+        if not id_cache:
+            self.logger.warning(
+                f'ID cache file not found {self.id_cache_filepath}')
```

Comment:
This message is a bit misleading. You will also get a Falsey response If your file exists but is empty.